modules/sdk-coin-xlm/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@bitgo/sdk-coin-xlm",
+  "version": "1.0.0",
+  "description": "BitGo SDK coin library for Xlm",
+  "main": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "build": "yarn tsc --build --incremental --verbose .",
+    "fmt": "prettier --write .",
+    "check-fmt": "prettier --check .",
+    "clean": "rm -r ./dist",
+    "lint": "eslint --quiet .",
+    "precommit": "yarn lint-staged",
+    "prepare": "npm run build",
+    "test": "npm run coverage",
+    "coverage": "nyc -- npm run unit-test",
+    "unit-test": "mocha"
+  },
+  "author": "BitGo SDK Team <sdkteam@bitgo.com>",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14 <17"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BitGo/BitGoJS.git",
+    "directory": "modules/sdk-coin-xlm"
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
+      "yarn prettier --write",
+      "yarn eslint --fix"
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "nyc": {
+    "extension": [
+      ".ts"
+    ]
+  },
+  "dependencies": {
+    "@bitgo/sdk-core": "^1.0.1",
+    "@bitgo/statics": "^6.17.0",
+    "@bitgo/utxo-lib": "^2.2.2",
+    "bignumber.js": "^8.0.1",
+    "lodash": "^4.17.14",
+    "stellar-sdk": "^10.0.1",
+    "superagent": "^3.8.3"
+  },
+  "devDependencies": {
+    "@bitgo/sdk-api": "^1.0.1",
+    "@bitgo/sdk-test": "^1.0.0"
```

Comment:
Are we doing @types/lodash in devDependencies for these still or relying on root?

Reply:
since it's specified in root we don't need to specify it here. We should clean up all the redundant dependencies as part of https://bitgoinc.atlassian.net/browse/BG-51022